guides/source/configuring.md

```diff
@@ -172,7 +172,7 @@ Rails allows you to alter what generators are used with the `config.generators`
 ```ruby
 config.generators do |g|
   g.orm :active_record
-  g.test_framework :test_unit
+  g.test_framework :minitest
```

Comment:
is test_framework option here a implementation thing @rafaelfranca ?
Maybe we should change a few places but not all, thoughts?